Gemfile

```diff
@@ -56,3 +56,6 @@ end
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 
 gem 'carrierwave'
+
+# 日本語化のために入れる
+gem 'i18n_generators'
```

Comment:
開発時だけ必要なものだと思います。

https://github.com/amatsuda/i18n_generators#installation
> Add 'i18n_generators' to your Gemfile, then bundle. Restricting the group to :development would be a good idea, since the generator would probably be used only in development mode.

とある通りですので対応しましょうー

Reply:
コメントありがとうございます！
すみません、何を間違えてしまっているのかがわからず...
記載の通りgemfileに 'i18n_generators'と記載し、bundleでインストール作業を行いました。
gemfileがあるということはbundleでインストールしたことがわかるのかなと思ったのですが、そうではないということでしょうか??

Reply:
> Restricting the group to :development would be a good idea,

このへんのことですね。Gemfileごらん頂くとわかりますが、
```ruby
group :development do
  gem 'listen', '~> 3.3'
  gem 'rack-mini-profiler', '~> 2.0'
  gem 'rubocop-fjord', require: false
  gem 'rubocop-rails', require: false
  gem 'spring'
  gem 'web-console', '>= 4.1.0'
end
```
という部分があるんです。このブロックで囲まれている部分は、本番環境(正確にはRAILS_ENVがproductionの状態ではインストールされません。なぜかというとこれらが開発時だけ必要なものだからです。今回適用gemにも同じことが言えませす。そして僕が引用したのはそのことに関する言及ですね。

Reply:
その部分だったのですね。
つまり`gem 'i18n_generators'`は開発時のみ必要なので記載する必要はなかった、
しかし、開発時には書かないと行けないので、下記でご指摘いただいていた.gitignoreにgemfileを追跡しないよう記載してからpushするべきだった、という理解でよろしいでしょうか？？

Reply:
そうじゃないです。英語の部分は「グループを開発に限定するのは良いアイデアだと思います。」と書かれてます。つまり、
```ruby
group :development do
  ...
  gem 'i18n_generators'
end
```
ってやりましょうってことです

Reply:
Restricting the group to :developmentってそういうことだったんですね...
gemfileの書き方にも色々あるとは知りませんでした。
よくみると上の方にgroup :development doがあったのでその中に仲間入りさせておきました。